stable/enterprise/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: enterprise
-version: "3.17.2"
+version: "3.17.3"
```

Comment:
not released yet. this is only going into our environment, so we dont bump it to 3.18.0 yet.